api/trace/noop_span.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+
+	"github.com/open-telemetry/opentelemetry-go/api/core"
+	"github.com/open-telemetry/opentelemetry-go/api/event"
+)
+
+// SpancContext returns an invalid span context.
+func (sp *noopSpan) SpanContext() core.SpanContext {
```

Comment:
Suggestion if this type is expected to conform to an interface declare that here via:

```go
var _ <Interface> = &noopSpan{}
```

Reply:
thx. Done.